Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1331,6 +1335,17 @@ class BrowserViewController: UIViewController {
                 // for reasons above about URL spoofing risk.
                 navigateInTab(tab: tab)
             }
+            
+            // Rewards reporting
+            if let url = change?[.newKey] as? URL, !url.isLocal {
+                // Notify Rewards of new page load.
+                if let rewardsURL = rewardsXHRLoadURL,
+                    url.host == rewardsURL.host {
+                    tabManager.selectedTab?.reportPageNaviagtion(to: rewards)
+                    // Not passing redirection chain here, in page navigation should not use them.
+                    tabManager.selectedTab?.reportPageLoad(to: rewards, redirectionURLs: [])
```

Comment:
URL kvo gets called before `didFinish` or `didReceiveServerRedirectForProvisionalNavigation`, 
it does not have access to redirection urls

Reply:
`reportPageNaviagtion` spelling error. Should be `reportPageNavigation`